tern/classes/image_layer.py

```diff
@@ -245,3 +246,24 @@ def gen_fs_hash(self):
             tar_file = rootfs.get_layer_tar_path(self.__tar_file)
             rootfs.extract_tarfile(tar_file, fs_dir)
             self.__fs_hash = rootfs.calc_fs_hash(fs_dir)
+
+    def add_files(self):
+        fs_path = rootfs.get_untar_dir(self.__tar_file)
+        hash_file = os.path.join(os.path.dirname(fs_path), self.__fs_hash) + '.txt'
+        pattern = re.compile(r'([\w\-|]+)\s+([\w\.\/]+)')
+        with open(hash_file) as f:
+            content = f.readlines()
+        for line in content:
+            m = pattern.search(line)
+            if m:
```

Comment:
The linter complains that there are too many local variables. I think you can use these values in the FileData assignment. Eg: `file_data = FileData(os.path.basename(m.group(2)), m.group(2))`